src/main/scala/caravan/bus/tilelink/Harness.scala

```diff
@@ -0,0 +1,79 @@
+package caravan.bus.tileink
+import caravan.bus.common.{AddressMap, BusDecoder, DeviceAdapter, Switch1toN}
+import chisel3._
+import chisel3.experimental.ChiselEnum
+import chisel3.stage.ChiselStage
+import chisel3.util.{Cat, Decoupled, MuxLookup}
+import chisel3.util.experimental.loadMemoryFromFile
+
+
+class DummyMemController(programFile: Option[String])(implicit val config: TilelinkConfig) extends Module {
```

Comment:
Let's take the `DummyMemController` to a common folder. There is already one declared in the wishbone package as well. We can then re-use them in our Harness.